src/Raven.Server/Documents/AI/AbstractChatCompletionClient.cs

```diff
@@ -325,6 +327,22 @@ private void HandleUnsuccessfulResponse(HttpResponseMessage response, BlittableJ
                     RequestId = reqId
                 };
         }
+
+        async Task<BlittableJsonReaderObject> GetErrorContent()
```

Comment:
check how it is done in `ReadExceptionFromServer`

Reply:
Done